<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ConnectDialog</class>
 <widget class="QWidget" name="ConnectDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Connect</string>
  </property>
  <property name="toolTip" >
   <string>Choose port box is connected to</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="layoutDirection" >
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="lineWidth" >
        <number>1</number>
       </property>
       <property name="text" >
        <string>&amp;Port:</string>
       </property>
       <property name="buddy" >
        <cstring>comboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Connect to Box</string>
       </property>
       <property name="text" >
        <string>&amp;Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>C&amp;ancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
